src/main/com/cqfn/degitx/simulator/Region.kt

```diff
@@ -0,0 +1,9 @@
+package com.cqfn.degitx.simulator
+
+/**
+ * Geography regions of server placement.
+ * ABC ordered.
+ */
+enum class Region {
+    Africa, Asia, Australia, China, EU, NorthAmerica, Russia, SouthAmerica
```

Comment:
@IlyaMoskva do we need to know all names of regions? Most probably we'll need to specify only the number of regions to simulate.

Reply:
@g4s8, Moreover, now regions is not used anywhere, I decided to omit it for MVP as not important. In the future, it should be number in settings, to be set during Graph generation.